src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,32 +2,39 @@
 
 public class DefaultCountingOutRhymer {
 
-    private int[] numbers = new int[12];
+    private static final int STACK_CAPACITY = 12;
+    private static final int EMPTY_INDEX = -1;
+    private static final int FULL_INDEX = 11;
```

Comment:
This constant is related to STACK_CAPACITY. Changing only one leads to an error (inconsistency).